api/README.md

```diff
@@ -208,20 +264,29 @@ const paymentInfo = await api.program.paymentInfo(alice);
 const transactionFee = paymentInfo.partialFee.toNumber();
 consolg.log(transactionFee);
 ```
+</details>
+
+
 
 ### Calculate gas for messages
+It's possible to find out the minimum gasLimit value to specify when sending extrinsic.
+The `gearApi.program.calculateGas.[method]` will help.
 
 Gas calculation returns GasInfo object contains 3 parameters:
 
 - `min_limit` - Minimum gas limit required for execution
 - `reserved` - Gas amount that will be reserved for some other on-chain interactions
 - `burned` - Number of gas burned during message processing
+- `may_be_returned` - value that can be returned in some cases
```

Comment:
Prolly worth to specify what cases

Reply:
It depends on contract. We cannot know it